quiz.py

```diff
@@ -1,6 +1,61 @@
 """
 File: quiz.py
-Author: Name
-Date: YYYY-MM-DD
-Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+Author: Garren Yu
+Date: 2024-03-01
+Description: a simple multiple-choice quiz game
+"""
+#header
+print("Multiple-Choice Quiz Game")
+
+#print Question 1 and get input from user
+print("\n1. longest length unit" + "\n(a) 12in" + "\n(b) 24.5cm" + "\n(c) 0.00013mi")
```

Comment:
Good use of `\n` special character for a new line. Note that you don't have to concatenate for new lines (no need to use `+`. Furthermore, to make this bit of code more readable, try using the `\` character to "escape" a line so that you can continue it on the next line. Combine this with indenting, and using your special characters, you can write the same output statement as:

```python
print("\n1. longest length unit \
      \n(a) 12in \
      \n(b) 24.5cm \
      \n(c) 0.00013mi")
```

Do you see how much easier it is to read that? You can "see" the actual output formatting without having to run the code. Try changing all of your output statements to be user-friendly like this.